consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/validation/RoundChangeMessageValidatorTest.java

```diff
@@ -227,35 +227,6 @@ public void prepareFromNonValidatorFails() {
     assertThat(messageValidator.validate(message)).isFalse();
   }
 
-  @Test
```

Comment:
What about testing that the round number in the block does not affect the validation?

Reply:
done